docs/concepts.md

```diff
@@ -126,6 +127,22 @@ For example, you can use params to ensure that you have 3 Redis replicas in your
 
 ---
 
+### Module
+
+Modules provide a way for you to share components across environments. More concisely, a module refers to a subdirectory in `components/` containing its own `params.libsonnet`.
+
+For example, `ks module create <module-name>` creates `components/<module-name>/params.libsonnet`.
+
+ksonnet will use the components defined in `components/` by default. If you want to use another set of components, modules provides a way to do this without initializing another ksonnet app.
+
+A module can:
+
+* used across multiple [environments](#environment) similar to components.
```

Comment:
Would this look better if `be` was prepended?

Reply:
Fixed